thirdparty/vars.sh

```diff
@@ -404,10 +404,10 @@ LLVM_SOURCE="llvm-project-16.0.6.src"
 LLVM_MD5SUM="dc13938a604f70379d3b38d09031de98"
 
 #clucene
-CLUCENE_DOWNLOAD="https://github.com/StarRocks/clucene/archive/refs/tags/starrocks-2024.06.03.tar.gz"
-CLUCENE_NAME="starrocks-clucene-2024.06.03.tar.gz"
-CLUCENE_SOURCE="starrocks-clucene-2024.06.03"
-CLUCENE_MD5SUM="377d04800b556b32e5d90ca68b219e47"
+CLUCENE_DOWNLOAD="https://github.com/StarRocks/clucene/archive/refs/tags/starrocks-2024.06.03-fix-arm.tar.gz"
+CLUCENE_NAME="starrocks-clucene-2024.06.03-fix-arm.tar.gz"
```

Comment:
fix-arm is not a good name and better remove

Reply:
Fixed